src/game/option_passport.c

```diff
@@ -207,18 +207,17 @@ void Option_Passport(INVENTORY_ITEM *inv_item)
                 g_InputDB = (INPUT_STATE) { 0 };
             }
         } else if (m_PassportMode == 0) {
-            if (g_InvMode == INV_DEATH_MODE) {
-                if (inv_item->anim_direction == -1) {
-                    g_InputDB = (INPUT_STATE) { 0, .left = 1 };
-                } else {
-                    g_InputDB = (INPUT_STATE) { 0, .right = 1 };
-                }
-            }
             if (!m_PassportText) {
                 if (g_InvMode == INV_TITLE_MODE
                     || g_CurrentLevel == g_GameFlow.gym_level_num) {
                     m_PassportText = Text_Create(
                         0, -16, g_GameFlow.strings[GS_PASSPORT_NEW_GAME]);
+                } else if (g_InvMode == INV_DEATH_MODE) {
+                    if (g_SavedGamesCount == 0) {
+                        g_InputDB.left = 0;
```

Comment:
What's this about?

Reply:
If there's no saved games, you can't turn the page left from restart level to the load game page.